fcrepo-audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -57,6 +57,7 @@
     static final String PROV = "http://www.w3.org/ns/prov#";
     static final String XSD = "http://www.w3.org/2001/XMLSchema#";
     static final String EVENT_TYPE = "http://id.loc.gov/vocabulary/preservation/eventType/";
+    static final String EVENT_NAMESPACE = "http://fedora.info/definitions/v4/event#";
```

Comment:
Because that would require an update to `fcrepo-camel` and a release of that artifact. I see the upcoming 4.6.0 release of fcrepo-camel-toolbox as a "bridge release": one that is compatible with 4.5.1 and before AND 4.6.0 and later; I would like the next release of `fcrepo-camel` to fully adopt the new serialization (and therefore be completely incompatible with older versions of Fedora). The 4.7.0 release of fcrepo-camel-toolbox would use that version of camel. At that point, the defn of this constant will go into fcrepo-camel.